crates/speedy-transform/src/web_transform/clear_mark.rs

```diff
@@ -0,0 +1,11 @@
+use swc_common::SyntaxContext;
+use {swc_ecma_ast::Ident, swc_ecma_visit::VisitMut};
+
+#[derive(Clone, Copy)]
+pub struct ClearMark;
+
+impl VisitMut for ClearMark {
+  fn visit_mut_ident(&mut self, ident: &mut Ident) {
```

Comment:
Requiring mutable reference is not necessary.

Reply:
[visit_mut_ident](https://docs.rs/swc_ecma_visit/latest/swc_ecma_visit/fn.visit_mut_ident.html)

sames that `self` must be `&mut`?

Reply:
LGTM